packages/realm-server/migrations/1724826419802_add-display-names-to-boxel-index.js

```diff
@@ -0,0 +1,11 @@
+exports.up = pgm => {
+  pgm.addColumns('boxel_index', {
+    display_names: 'jsonb',
```

Comment:
Since I computed the card type summary in the batch's done function, I need to store the display name in `boxel_index` so I can retrieve the card type summary fields by querying `boxel-index`. I followed the approach we use for `types`, so I made `display_names` a JSONB field.

Reply:
why can't you just query the `realm_meta` table in done()?

Reply:
ok, i misunderstood. you run the card to get the display names for the entire adoption chain in current-run, then we store those names in the boxel_index, and specifically that data is then fed into realm_meta. this seems like a fine approach.